tooling/templatize/testdata/pipeline.yaml

```diff
@@ -2,15 +2,15 @@ serviceGroup: Microsoft.Azure.ARO.Test
 rolloutName: Test Rollout
 resourceGroups:
 - name: {{ .regionRG }}
-  subscription: {{ .serviceClusterSubscription }}
```

Comment:
if you change this, you also need to change testdata/config.yaml

Reply:
AH, my bad I used the config from the root folder. Let me revert that